swift-service/src/main/java/com/facebook/swift/service/ThriftServiceProcessor.java

```diff
@@ -149,6 +149,7 @@ public void onSuccess(Boolean result)
                         @Override
                         public void onFailure(Throwable t)
                         {
+                            LOG.error("Failed to process method [" + method.getName() + "] of service [" + method.getServiceName() + "]", t);
```

Comment:
ThriftMethodProcessor catches exception thrown by service handler and writes exception information into a field "exception" of reply message. Which treated by ThriftServiceProcessor as success (indeed, at this point it sees a normal reply message with a field "exception").
So this change will not produce logs for service method throwing application exception.